tests/stack/test_api.py

```diff
@@ -552,15 +541,23 @@ def test_deploy_config(self, stack_api, tmpdir):
                         f.write("print('test')\n")
 
         new_stack_status_1 = stack_api.deploy(test_stack)
-        # new stack status should have an identical list of "resources"
-        assert new_stack_status_1.get(api.STACK_RESOURCES) == test_stack.get(api.STACK_RESOURCES)
-        for deployed_resource in new_stack_status_1.get(api.STACK_DEPLOYED):
-            # All functions to pull the deployed output were mocked to return deploy_output
-            assert deployed_resource.get(api.RESOURCE_DEPLOY_OUTPUT) == test_deploy_output
+        new_databricks_ids_1 = [deployed_resource.get(api.RESOURCE_DATABRICKS_ID)
```

Comment:
It looks like this test only verify the databricks_ids. Could you make it verify the format of the entire new_stack_status_1?

Reply:
done